### ISSUE TITLE
Is there a way to define this card is debit or credit card 

### ISSUE THREAD

**Issue by abdelrahmanahmed** [PawelDecowski/jquery-creditcardvalidator#112]
How can you tell if any card entered is debit or credit card ?

**Comment by PawelDecowski** [comment]
Don’t think you can, based on the card number alone.

**Comment by abdelrahmanahmed** [comment]
then how did they identify card type [here](https://binlist.net/) based on number only?

**Comment by PawelDecowski** [comment]
That looks super cool but, unfortunately, they don’t say anything about the source for their data so I can’t say how reliable it is.

I’m yet to find a reliable source for [IIN](https://en.wikipedia.org/wiki/Payment_card_number#Issuer_identification_number_.28IIN.29)s that includes granular data such as whether it’s a debit or credit card.

Even they say on that page:

> Although this service is very accurate, don't expect it to be perfect.

I wish they had anything to support their statement about it being very accurate.

The bottom line is, IIN list is not public so nobody has access to the full list except for financial institutions. The IIN lists that are out there are probably compilations based on debit and credit card numbers found “in the wild” so they’re far from accurate.
